install

```diff
@@ -145,6 +145,10 @@ def chgrp?(path)
   !File.grpowned?(path)
 end
 
+def user
+  @user ||= ENV["USER"] || `id -un`.chomp
```

Comment:
How about not making this a function but instead doing
```ruby
ENV["USER"] || `id -un`.chomp
```
so subprocesses can benefit?